packages/remirror__extension-entity-reference/src/entity-reference-extension.ts

```diff
@@ -169,6 +169,34 @@ export class EntityReferenceExtension extends MarkExtension<EntityReferenceOptio
     };
   }
 
+  /**
+   * Dispatch a transaction that selects the range of the entity reference then scrolls to it.
+   *
+   * @param entityReferenceId - The entity's reference Id.
+   *
+   * @returns True if the scrolling was applied, else it returns false
+   *
+   */
+  @command()
+  scrollToEntityReference(entityReferenceId: string): CommandFunction {
+    return ({ tr, dispatch }) => {
+      const entityReference = this.store.helpers.getEntityReferenceById(entityReferenceId);
+
+      if (!entityReference) {
+        return false;
+      }
+
+      const { doc } = tr;
+      const resolvedFrom = doc.resolve(entityReference.from);
+      const resolvedTo = doc.resolve(entityReference.to);
+      const entityReferenceSelection = new TextSelection(resolvedFrom, resolvedTo);
```

Comment:
```suggestion
      const entityReferenceSelection = TextSelection.between(resolvedFrom, resolvedTo);
```

https://prosemirror.net/docs/ref/#state.TextSelection^between

use `TextSelection.between` so that ProseMirror can fix the position if `$from` and `$to` are a bit wrong.